styles/Microsoft/Accessibility.yml

```diff
@@ -0,0 +1,25 @@
+extends: existence
+message: "Don't use language (such as '%s') that defines people by their disability."
+link: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms
+level: error
+ignorecase: true
+tokens:
+  - a victim of
+  - able-bodied
+  - affected by
+  - an epileptic
+  - crippled
+  - disabled
+  - dumb
+  - handicapped
+  - handicaps
+  - healthy
+  - lame
+  - maimed
+  - missing a limb
+  - mute
+  - normal
```

Comment:
As with other things, my only perception of "normal" as a negative or derogatory thing is in reference to people (culture, ability, etc). The behaviour of an application is something I personally feel comfortable classifying as "normal" and "abnormal". However, that's my _personal_ sense of it, I've asked another person and they also felt that so long as it wasn't in reference to a person it didn't seem bad, and avoiding it would complicate the language (with the adverse effect of accidentally making the written text _less_ accessible, from a general linguistic perspective).

"Normal" also doesn't _immediately_ evoke the same kinds of historical violence when not used to describe a person. Whereas other terms like "master/slave", "nuke", etc, directly and unambiguously evoke tremendous violence, regardless of the context. It makes perfect sense to avoid those, especially because there are actually _clearer_ terms to use. Same with blacklist/whitelist (deny/allow is much clearer).

The rule message at the top of this file is:

> message: "Don't use language (such as '%s') that defines people by their disability."

We have only a handful of documents that this would ever be relevant to (where we talk about people and their behaviour), namely the decision-making process and the code of conduct. I get the sense that Microsoft created this particular rule for non-technical documentation, like management or HR documents? There are probably some software domains this is relevant to, not sure Openverse fits into that domain. Maybe HR software, education software, hospital software, etc?

That's just my 2-cents. I don't mind changing the words we use, especially because "inactive" and "deactivated" are just as clear as "disabled" if used in the right contexts. "Standard" feels like less of an unambiguous alternative to "normal", especially because in software it evokes _standards_, but doesn't feel sufficiently different for me to object to changing it if someone feels strongly about it not being used _ever_. Though my sense of the disability activism space is that no one would care about using "normal" to talk about the behaviour of a computer program. I also have a hard time imagining that a derogatory/negative usage of the word would get past our code review process, considering the concern and care Openverse maintainers have historically had for a wide range of other issues.

So anyway, I agree that we could remove this rule altogether. Its description of itself shows that it's not really for software documentation. On the other hand, if someone feels strongly that we should add it, fine by me as well. Maybe we can remove it for now to minimise the changes in this PR and get input from other maintainers afterwards, and make appropriate adjustments as needed?

Reply:
Oh, of course, the other option is to leave the rule, but remove the words you've noted are common in software without a negative connotation: normal, disabled, healthy, unhealthy. Can't hurt to have an automatic check for the other words here.